### PR TITLE
Seedtag Bid Adapter: read video params from mediaTypes, and allow override from bidder params, support passing adomain

### PR DESCRIPTION
**Pull request by ybootin**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

Supports video params from adunit and bidder : https://github.com/prebid/Prebid.js/issues/6512
Support advertiserDomains : https://github.com/prebid/Prebid.js/issues/6650

### CODE REVIEW

**Comment by patmmccann** [comment]
Did you also fix #6650 ?

**Comment by ybootin** [comment]
@patmmccann thx for the reminder, i've made an update to include it in this fix

File: modules/seedtagBidAdapter.js [hunk 1]
```diff
@@ -58,13 +58,12 @@ function hasMandatoryParams(params) {
...
-  const isPlayerSize =
-    !!utils.deepAccess(mediaTypes, 'video.playerSize') &&
-    utils.isArray(utils.deepAccess(mediaTypes, 'video.playerSize'));
-  return isVideoInStream && isPlayerSize;
+function hasMandatoryVideoParams(bid) {
+  const videoParams = getVideoParams(bid)
+
+  return hasVideoMediaType(bid) && !!videoParams.playerSize &&
+    utils.isArray(videoParams.playerSize) &&
+    videoParams.playerSize.length > 0;
```
Comment (patmmccann) [inline modules/seedtagBidAdapter.js:66]: do you want to treat h and w as a substitute for playersize?

File: modules/seedtagBidAdapter.js [hunk 2]
```diff
@@ -114,7 +128,10 @@ function buildBid(seedtagBid) {
     netRevenue: true,
     mediaType: mediaType,
     ttl: seedtagBid.ttl,
-    nurl: seedtagBid.nurl
+    nurl: seedtagBid.nurl,
+    meta: {
+      advertiserDomains: seedtagBid && seedtagBid.adomain && seedtagBid.adomain.length > 0 ? seedtagBid.adomain : []
```
Comment (patmmccann) [inline modules/seedtagBidAdapter.js:133]: i think should probably be `seedtagBid.adomain && seedtagBid.adomain.length > 0 ? seedtagBid.adomain : []`

**Review by ChrisHuie — APPROVED** [review]

**Pull request merged**